skyportal/utils/cosmology.py

```diff
@@ -0,0 +1,64 @@
+from astropy import cosmology
+from astropy import units as u
+
+from baselayer.log import make_log
+
+log = make_log('cosmology')
+
+
+def establish_cosmology(cfg={}, fallback_cosmology=cosmology.Planck18_arXiv_v2):
```

Comment:
Why have a fallback cosmology defined here and not in `config.yaml.defaults`?

Reply:
Because I dont want to the cosmology instantiation to fail if the user makes a mistake.

Reply:
I thought more about this, and am cool with it. Astropy does a similar thing sometimes, with warnings. As long as it is clear in the logs.